app/src/main/java/com/github/h3lp3rs/h3lp/HelpeeSelectionActivity.kt

```diff
@@ -31,24 +34,15 @@ import kotlin.collections.ArrayList
 const val EXTRA_NEEDED_MEDICATION = "needed_meds_key"
 const val EXTRA_CALLED_EMERGENCIES = "has_called_emergencies"
 const val EXTRA_EMERGENCY_KEY = "emergency_key"
-const val EXTRA_HELPEE_ID = "helpee_id"
 
 /**
  * Activity in which the user can select the medications they need urgently
  */
-class HelpParametersActivity : AppCompatActivity() {
-    // userLocation contains the user's current coordinates (is initialized to null since we could
-    // encounter an error while getting the user's location)
-
-    private var meds: ArrayList<String> = ArrayList()
-    private var skills: HelperSkills? = null
-    private val currentTime: Date = Calendar.getInstance().time
+class HelpeeSelectionActivity : AppCompatActivity() {
```

Comment:
Although the name makes sense, it sounds like the activity is meant to selesct an Helpee